src/main/java/io/vertx/ext/mail/MailConfig.java

```diff
@@ -290,53 +300,33 @@ public MailConfig setPassword(String password) {
     return this;
   }
 
-  /**
-   * get whether ssl is used on connect
-   *
-   * @return ssl option
-   */
-  public boolean isSsl() {
-    return ssl;
-  }
-
-  /**
-   * Set the sslOnConnect mode for the connection.
-   *
-   * @param ssl true is ssl is used
-   * @return a reference to this, so the API can be used fluently
-   */
-  public MailConfig setSsl(boolean ssl) {
-    this.ssl = ssl;
+  // Maintain compatibility of return type
```

Comment:
Keeps existing code that fluently chains this call as a `MailConfig` from breaking.